### ISSUE TITLE
error: member function expected 0 argument(s) when building

### ISSUE THREAD

**Issue by tyjak** [shtanton/gmi2html#22]
I clone the project on thet latest version on master branch (84cfa53c219dc7119b1716368cd6d8ed6dabc867)
zig version : 0.10.1
I'm on archlinux x86_64 Linux 6.2.12-arch1-1

The error : 
```
/home/david/src/gmi2html/src/main.zig:208:16: error: member function expected 0 argument(s), found 1
    while (args.next(allocator)) |maybeArg| {
           ~~~~^~~~~
/usr/lib/zig/std/process.zig:777:9: note: function declared here
    pub fn next(self: *ArgIterator) ?([:0]const u8) {
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    callMain: /usr/lib/zig/std/start.zig:614:32
    initEventLoopAndCallMain: /usr/lib/zig/std/start.zig:548:51
    remaining reference traces hidden; use '-freference-trace' to see all reference traces

error: gmi2html...
error: The following command exited with error code 1:
/usr/bin/zig build-exe /home/david/src/gmi2html/src/main.zig -fno-strip --cache-dir /home/david/src/gmi2html/zig-cache --global-cache-dir /home/david/.cache/zig --name gmi2html -fsingle-threaded --enable-cache
error: the following build command failed with exit code 1:
```

**Comment by shtanton** [comment]
I had a quick look and the type of `ArgIterator.next` changed with zig 0.10.0.

I don't have time to fix right now but will push an updated version in the next few days.

If anyone else wants to make whatever small changes are needed to bring it up to date, send a pull request and I'll merge it.

Until then, are you able to compile it with 0.9.x?

**Comment by shtanton** [comment]
Should be fixed now, I've released version 0.5.0 which is compatible with zig 0.10.0 and 0.10.1.

AUR has been updated too.
